src/screens/Account/TokenContractAddress.js

```diff
@@ -25,6 +25,9 @@ const forceInset = { bottom: "always" };
 
 const TokenContractAddress = ({ account, onClose, url, token }: Props) => {
   const { colors } = useTheme();
+
+  const currencyId = account ? account.token.name : token ? token.name : "";
```

Comment:
This helper helps.

https://github.com/LedgerHQ/ledger-live-common/blob/2eb8c4f3bb8dc1adee636752750d2e8c01d06482/src/account/helpers.js#L54

Reply:
not relevant